docs/_articles/en/codebuilder/cb-start.md

```diff
@@ -0,0 +1,225 @@
+
+The metadata is here for your reference:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+   <fullName>createdon__c</fullName>
+   <description>Date of Account Creation</description>
+   <externalId>false</externalId>
+   <label>Created On</label>
+   <required>false</required>
+   <trackFeedHistory>false</trackFeedHistory>
+   <type>Date</type>
+</CustomField>
+```
+We’ll now make a simple edit to this field and deploy our changes back to our org with a single click.
+
+1. Edit `createdon_c.field-meta.xml` and change the `<required>` tag value to `true` to indicate that this custom field is required. 
+2. Right click the `objects/Account` folder and click **SFDX: Deploy Source to Org**. 
+3. After the command has successfully run, go back to your org and check details of the **Created On** custom field and confirm that it’s now a required field. 
+
+## Create and Deploy a New Lightning Web Component
+For another exercise in deploying, let’s learn how to quickly create a simple Lightning Web Component in our Code Builder project. We’ll then deploy this component to our org using a single command. Here we go:
+
+Let’s create a Lightning Web Component – 
+
+1. Press Ctrl+Shift+P (⇧⌘P) run **SFDX: Create Lightning Web Component** to create a New Lightning Web Component. 
+2. Give the component a name, say, `newCBComponent`.
+3. Press **Enter** to accept the default file location (`force-app/main/default/lwc`).
+4. Press **Enter**.
+
+Three new files are created in the `force-app/main/default/lwc/newCBComponent` folder.
+
+Update the files –
+
+1. In the HTML file, `newCBComponent.html`, copy and paste the following code:
+
+```
+<template>
+    <p>Hello, {greeting}!</p>
+    <lightning-input label="Name" value={greeting} onchange={handleChange}></lightning-input>
+</template>
+```
+2. In the HTML file, `newCBComponent.js`, copy, and paste the following code:
+
+```
+import { LightningElement } from 'lwc';
+
+export default class NewCBComponent extends LightningElement {
+   greeting = 'Great going learning Code Builder!';
+   changeHandler(event) {
+       this.greeting = event.target.value;
+   }
+}
+```
+3. In the HTML file, `newCBComponent.js-meta.xml`, copy and paste the following code:
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+   <apiVersion>54.0</apiVersion>
+   <isExposed>true</isExposed>
+   <targets>
+   <target>lightning__AppPage</target>
+   <target>lightning__RecordPage</target>
+   <target>lightning__HomePage</target>
+ </targets>
+</LightningComponentBundle>
+```
+4. Save all the files.
+
+Let’s deploy this new component to our org – 
+
+1. Right-click the `force-app/main/default/lwc/newCBComponent` folder and click **SFDX: Deploy Source to Org** 
+
+Your output window shows this message:
+
+```
+
+=== Deployed Source
```

Comment:
This doesn't look right in preview either but could be fine in real output. (It's wrapping)

Reply:
I think this is also an issue with netlify. I checked how it looks in VS Code preview.